src/java/htsjdk/samtools/fastq/FastqRecord.java

```diff
@@ -23,62 +23,144 @@
  */
 package htsjdk.samtools.fastq;
 
+import htsjdk.samtools.Read;
+import htsjdk.samtools.SAMUtils;
+import htsjdk.samtools.util.StringUtil;
+
 import java.io.Serializable;
 
 /**
- * Represents a fastq record, fairly literally, i.e. without any conversion.
+ * Simple representation of a FASTQ record, without any conversion
  */
-public class FastqRecord implements Serializable {
+public class FastqRecord implements Read, Serializable {
     private static final long serialVersionUID = 1L;
-    private final String seqHeaderPrefix;
-    private final String seqLine;
-    private final String qualHeaderPrefix;
-    private final String qualLine;
-
-    public FastqRecord(final String seqHeaderPrefix, final String seqLine, final String qualHeaderPrefix, final String qualLine) {
-        if (seqHeaderPrefix != null && !seqHeaderPrefix.isEmpty()) this.seqHeaderPrefix = seqHeaderPrefix;
-        else this.seqHeaderPrefix = null;
-        if (qualHeaderPrefix != null && !qualHeaderPrefix.isEmpty()) this.qualHeaderPrefix = qualHeaderPrefix;
-        else this.qualHeaderPrefix = null;
-        this.seqLine = seqLine ;
-        this.qualLine = qualLine ;
+    private final String readName;
+    private final String readString;
+    private final String qualityHeader;
+    private final String qualityString;
+
+    /**
+     * Default constructor
+     *
+     * @param readName      the read name
+     * @param readStrig     the sequence string
```

Comment:
Typo in read**Strig**.  Also, could you make clear that these are the bases?  I feel like sequence can be interpreted in a number of ways